scripts/TetToQuadConnection/quadMeshingFunctions.py

```diff
@@ -0,0 +1,110 @@
+# A python script meant to run the process of taking a tet mesh and creating a quad mesh of one of the boundaries.
+# The resulting vtk file path can be returned, along with a list of points and quads from the vtk file.
+
+import sys
+import subprocess
+from pathlib import Path
+import numpy as np
+SCRIPT_DIR = Path(__file__).resolve().parent
+if str(SCRIPT_DIR) not in sys.path:
+    sys.path.insert(0, str(SCRIPT_DIR))
+SWEEPS_DIR = SCRIPT_DIR.parent.parent
+
+path_to_api = Path(__file__).parent.parent.parent / "build" / "src" / "api"
+sys.path.insert(0, str(path_to_api))
+import sweeps
+
+def runQuadriflowProgram(inputFile: str):
+    """
+    Runs the quadriflow program to create a quad mesh from a tet mesh input file.
+    inputFile: str - Path to the tet mesh file.
+    """
+    quadriflowExecutable = SWEEPS_DIR / "deps" / "ScaleUntrim" / "build" / "quadriflow"
+    configPath = SWEEPS_DIR / "deps" / "ScaleUntrim" / "setting.config"
+    
+    if not quadriflowExecutable.exists():
+        raise FileNotFoundError(f"C++ executable not found at {quadriflowExecutable}")
+    
+    
+    command = [str(quadriflowExecutable), inputFile, configPath]
+
+    try:
+        result = subprocess.run(command, check=True, capture_output=True, text=True)
+        print("OUTPUT:")
+        print(result.stdout)
+    except subprocess.CalledProcessError as e:
+        print(f"Error running command: {e.cmd}")
+        print(f"Return code: {e.returncode}")
+        print(f"OUTPUT: {e.output}")
+        print("END OF OUTPUT MESSAGE")
+        print(f"ERROR OUTPUT: {e.stderr}")
+        print("END OF ERROR MESSAGE")
+        sys.exit(1)
+    except Exception as e:
+        print(f"An error occurred: {e}")
+        sys.exit(1)
+
+def generateQuadMesh(tetMesh: str):
```

Comment:
This is nitpicky, but the input here is a tri mesh, not a tet mesh.  Could be good for avoiding later confusion if you renamed that here.